### PR TITLE
update instructions

### PR DESCRIPTION
**Pull request by Erin-Cecele**
Thank you for contributing to the Quire Documentation & Website! Please complete the form below to submit your pull request for review.

*For the Title of this pull request, please use the format "Type/Issue-#: Brief description." For Type, the options are Edit, Add, Translate. Issue-# is only needed if this pull request addresses an existing issue.*

### Checklist 

Please put an X within the brackets that apply `[X]`.

- [X] I have read the [CONTRIBUTING.md](https://github.com/thegetty/quire/blob/main/CONTRIBUTING.md) file.

- [X] I have made my changes in a new branch and not directly in the main branch

- [X] I am requesting feedback on a draft pull request

### Is this pull request related to an open issue? If so, what is the issue number?

#558 

### Please describe the goal of this pull request and the changes that were made.

I updated the instructions to the best of my abilities. I tested the manual Netlify Deploy section, and that is working correctly. I updated most of the Continuous Deploy section but did not test it. I wasn't sure if step # 2 in this section (`package.json` and `netlify.toml`) could be eliminated. 

I updated what I could in the GitHub Pages section, but I wasn't sure how best to update steps 2, 4, 5, etc. because of the new project structure. I know you said there isn't likely to be a `bin/deploy.sh` script anymore either.

### CODE REVIEW

**Comment by geealbers** [comment]
@Erin-Cecele, I updated both the Deploy and Multiformat Output pages as best I could at the moment. Take a look when you can and then hopefully we can merge these in soon and get them out to folks. 

Thanks for getting it started oh so long ago!

File: content/docs-v1/multiformat-output.md [hunk 1]
```diff
@@ -13,40 +11,27 @@ Quire is designed to create a website version, a PDF version and two e-book vers
...
+Create the HTML files for your project by running `quire build` in your command-line shell. The files will be built into your project’s `_site` folder along with all the necessary static assets like image files, stylesheets and script files. The `_site` file will be updated and overwritten each time you run `quire build`.
 
-{{< q-class "box tip" >}}
-- If you are including PDF and e-book downloads as part of your online site (this is Quire’s default) you’ll need to update those files by running `quire epub` and `quire pdf` as described below **before** running `quire site`.
-{{< /q-class >}}
-
-You can hide specific pages of your project from the site output by adding `online: false` to the page YAML. Or conversely, you can add pages exclusively to the site output, by adding `epub: false` and `pdf: false` to the page YAML and leaving the `online` attribute unset, or set to `true`.
+If you are including PDF and e-book downloads as part of your online site (this is Quire’s default) you’ll need to update those files by running `quire epub` and `quire pdf` as described below **before** running `quire build`.
```
Comment (Erin-Cecele) [inline content/docs-v1/multiformat-output.md:16]: Oh, I didn't realize you had to run `quire pdf` and `quire epub` **before** `build`. Is this new? Or did I just misunderstand Matthew before? Does this mean if you were creating both a PDF and E-book the process would be: 

quire pdf
quire build
quire epub
quire build

File: content/docs-v1/multiformat-output.md [hunk 2]
```diff
@@ -13,40 +11,27 @@ Quire is designed to create a website version, a PDF version and two e-book vers
...
 
-Unlike `epub: false` and `pdf: false`, adding `online: false` does not stop a page from being built and included in the site output. Rather, it just unlinks those pages and adds a metadata tag to them to prevent them from being indexed by search engines. You may want to manually delete the pages from the `site` folder after outputting and before uploading to a web server if you want to be fully certain they can’t be accessed online under any circumstances.
+Links to download the EPUB, MOBI, and PDF versions are automatically included in the sidebar menu of the online version of your project. These links can be removed or modified by making changes to the `resource_link` information in your `content/_data/publication.yaml` file. You can add links to these files from *anywhere* in your markdown files by linking to `_assets/downloads/output.epub`, `_assets/downloads/output.mobi`, and `_assets/downloads/output.pdf` respectively.
```
Comment (Erin-Cecele) [inline content/docs-v1/multiformat-output.md:18]: I'm not clear on what you mean by "You can add links to these files from *anywhere* in your markdown files by linking to `_assets/downloads/output.epub`, `_assets/downloads/output.mobi`, and `_assets/downloads/output.pdf` respectively".

File: content/docs-v1/multiformat-output.md [hunk 3]
```diff
@@ -13,40 +11,27 @@ Quire is designed to create a website version, a PDF version and two e-book vers
...
-
-For developers creating custom templates, you can use templating logic to create specific outputs for the e-books by querying `if eq .Site.Data.params.epub true`. This parameter is set in the `config/epub.yaml` file.
+Quire outputs {{< q-def "reflowable" >}} e-books in the EPUB file format. EPUB is the most widely used format and will work on most devices and for most e-book vendors. EPUB is an official specification of the World Wide Web Consortium (W3C), and Quire outputs the latest version: EPUB 3.2.
 
 ### Create and View the E-Book Files
 
-Create an EPUB of your project by running `quire epub` in your command-line shell. An `output.epub` file will be created and saved to your project’s `static/downloads/` folder. This file will be updated and overwritten each time you run `quire epub`.
+When creating an EPUB, first run `quire build` in your command-line shell to generate files with your latest changes. Next run `quire epub`. An `epubjs.epub` file will be created and saved to your project’s main directory. This file will be updated and overwritten each time you run `quire epub`.
```
Comment (Erin-Cecele) [inline content/docs-v1/multiformat-output.md:30]: Here it says that you first run `build` and then `epub` which contradicts the section above unless I'm misreading/misunderstanding.

File: content/docs-v1/multiformat-output.md [hunk 4]
```diff
@@ -13,40 +11,27 @@ Quire is designed to create a website version, a PDF version and two e-book vers
...
 
-EPUB files can be viewed on the default Books app on macOS, or on a number of free EPUB readers available for both Windows and Mac.
+To include the EPUB file as a download from your online edition, mvove the `epubjs.epub` file into `content/_assets/downloads/` directory. Be sure to update the the `resource_link` information in your `content/_data/publication.yaml` file as described above.
```
Comment (Erin-Cecele) [inline content/docs-v1/multiformat-output.md:32]: typo "move"

**Review by Erin-Cecele — COMMENTED** [review]
@geealbers Multiformat output looks good to go, but I have some questions about site-deploy that we can maybe talk about in our afternoon meeting. Thank you!

File: content/docs-v1/multiformat-output.md (hunk 1 above)
Comment (geealbers) [reply content/docs-v1/multiformat-output.md:16]: Oh, shoot. Well this is problematic. To build the pdf, you were right, you have to run `quire build` and then `quire pdf` However, if you want to include the PDF as part of the built site (as a download) you have to move the PDF file you just created into your `contents/_assets/downloads` directory and then run `quire build` again. That's ... not great.

1. quire build
2. quire pdf
3. quire epub
4. Move the PDF and EPUB files
5. quire build

Ooof.

File: content/docs-v1/multiformat-output.md (hunk 2 above)
Comment (geealbers) [reply content/docs-v1/multiformat-output.md:18]: I guess I meant that you could have a sentence on a markdown page like "Download the free PDF and EPUB versions of this book.". I can clarify that in the write-up, or should I just delete that part?

**Comment by geealbers** [comment]
@Erin-Cecele, just pushed an update with some clarifications. Especially around adding PDF and EPUB files into the project for download. Let me know if they work

**Review by Erin-Cecele — COMMENTED** [review]
Love the way you reworded things in the multiformat section. Very clear! Just realized I can't approve since I'm the original PR author. But I approve!

**Comment by geealbers** [comment]
@Erin-Cecele Great, I approve too! Go ahead and merge anytime you're ready. thanks

**Pull request merged**